### ISSUE TITLE
Embed map on page

### ISSUE THREAD

**Issue by StephMSlack** [MantaRayMedia/bgs#313]
# Done criteria
- [ ] Error message received when trying to embed google map
![Screen Shot 2018-08-07 at 11.45.38.png](https://images.zenhubusercontent.com/59c3dc2bb0222d5de4789f2d/ad01a2e7-7c05-46b3-8902-41c63d3efcf4)

Sample on https://www.bgs.org.uk/events/2019-spring-meeting-2019

**Comment by dilwarhussain** [comment]
@StephMSlack This is fixed now. I'm not sure why the embed code wasn't working, but recreating another iframe nugget seemed to have fixed it.

**Comment by StephMSlack** [comment]
@dilwarhussain I'm trying to add the map on this page 
https://www.bgs.org.uk/events/se-sw-thames-joint-autumn-meeting-2018. I removed the iframe nugget. Saved the page. then edited the page and added a new iframe nugget and still got the error message. 
the Google iframe code is <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d9934.0056881784!2d-0.1153927535829936!3d51.50401623304107!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x487604b863c251b3%3A0x4e14a43e59b8dd29!2sGuy&#39;s+and+St+Thomas&#39;+Education+Centre!5e0!3m2!1sen!2suk!4v1534258937758" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe> and I am removing everything before https: and everything after 37758. 

I have managed to add the maps to other pages, but seem to be having issues with this one page!
